dev/com.ibm.ws.logging.osgi/src/com/ibm/ws/logging/internal/osgi/stackjoiner/bci/ThrowableClassFileTransformer.java

```diff
@@ -20,31 +20,30 @@
 
 public class ThrowableClassFileTransformer implements ClassFileTransformer {
 
-    @Override
-    public byte[] transform(ClassLoader loader, String className, Class<?> classBeingRedefined, ProtectionDomain protectionDomain,
-                            byte[] classfileBuffer) throws IllegalClassFormatException {
-        byte[] ba = null;
-        try {
-            ba = transformClassIfThrowable(classfileBuffer, className);;
-        } catch (Exception e) {
-
-            throw e;
-        }
-        return ba;
-    }
-
-    @SuppressWarnings("static-access")
+	@Override
+	public byte[] transform(ClassLoader loader, String className, Class<?> classBeingRedefined, ProtectionDomain protectionDomain,
+	                        byte[] classfileBuffer) throws IllegalClassFormatException {
+	    byte[] ba = null;
+	    try {
+	        ba = transformClassIfThrowable(classfileBuffer, className);;
+	    } catch (Exception e) {
+	
+	        throw e;
+	    }
+	    return ba;
+	}
+	
+	@SuppressWarnings("static-access")
 	private byte[] transformClassIfThrowable(byte[] cBuffer, String nameOfClass) {
-        ClassReader reader = new ClassReader(cBuffer);
-        ClassWriter writer = new ClassWriter(reader, ClassWriter.COMPUTE_FRAMES);
-        ClassVisitor visitor = writer;
-        /*
-        if (nameOfClass.equals("java/lang/Throwable")) {
-            visitor = new ThrowableClassAdapter(visitor);
-        }*/
-        visitor = new ThrowableClassAdapter(visitor);
-        reader.accept(visitor, reader.SKIP_FRAMES);
-        return writer.toByteArray();
-    }
+	    ClassReader reader = new ClassReader(cBuffer);
+	    ClassWriter writer = new ClassWriter(reader, ClassWriter.COMPUTE_FRAMES);
+	    /*
+	    if (nameOfClass.equals("java/lang/Throwable")) {
+	        visitor = new ThrowableClassAdapter(visitor);
+	    }*/
```

Comment:
remove comments for code that was never shipped

Reply:
Fixed in [6d1664b](https://github.com/OpenLiberty/open-liberty/pull/16276/commits/6d1664bfbbc9a0427d18ac11d9ec4f26ce3c4373)